pyomo/dataportal/parse_table_datacmds.py

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
This file is autogenerated by PLY and I believe should not be committed to the repo.  I believe there are issues if the table is out of date (wrt the version of PLY that the user has installed) and the user cannot edit the file (e.g., Pyomo was installed into the system python).